docs-source/content/security/rbac.md

```diff
@@ -132,17 +132,18 @@ the following `ClusterRoleBinding` entries are mapped to a `ClusterRole` grantin
 
 | Cluster Role Binding | Mapped to Cluster Role | Resource Access | Notes |
 | --- | --- | --- | --- |
-| Operator `general` | Operator `general` | **Read**: namespaces | [^1] |
+| Operator `general` | Operator `general` | **Read**: namespaces | [^3] |
 | | | **Edit**: customresourcedefinitions, ingresses, persistentvolumes | |
 | | | **Update**: domains (weblogic.oracle), domains/status | |
 | | | **Create**: tokenreviews, subjectaccessreviews, localsubjectaccessreviews, selfsubjectaccessreviews, selfsubjectrulesreviews | |
 | Operator `nonresource` | Operator `nonresource` | **Get**: /version/* | [^1] |
-| Operator `discovery` | Kubernetes `system:discovery` | **See**: [Kubernetes Discovery Roles](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#discovery-roles) | [^1] |
-| Operator `auth-delegator` | Kubernetes `system:auth-delegator` | **See**: [Kubernetes Component Roles](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#other-component-roles) | [^1] |
+| Operator `discovery` | Kubernetes `system:discovery` | **See**: [Kubernetes Discovery Roles](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#discovery-roles) | [^3] |
+| Operator `auth-delegator` | Kubernetes `system:auth-delegator` | **See**: [Kubernetes Component Roles](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#other-component-roles) | [^3] |
 
 
 [^1]: The binding is assigned to the operator `ServiceAccount`.
 [^2]: The binding is assigned to the operator `ServiceAccount`
       in each namespace listed with the `domainNamespaces` setting.
       The `domainNamespaces` setting contains the list of namespaces
       that the operator is configured to manage.
+[^3]: The binding is assigned to the operator `ServiceAccount`. In addition, the Kubernetes RBAC resources that the operator installation actually set up will be adjusted based on the value of the `dedicated` setting. By default,  the `dedicated` value is set to `false`, those security resources are created as `ClusterRole` and `ClusterRoleBindings`. If the `dedicated` value is set to `true`,  those will be created as `Roles` and `RoleBindings` in the namespace of the operator.
```

Comment:
those will be created -> those resources will be created

Reply:
done